plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/GroupMetadataServiceImpl.java

```diff
@@ -42,11 +48,29 @@
 {
   private final NpmGroupRepository npmGroupRepository;
 
+  private final AlphanumComparator comparator;
+
+  private boolean mergeMetadata = SystemPropertiesHelper.getBoolean(
+      "com.bolyuba.nexus.plugin.npm.service.internal.GroupMetadataServiceImpl.mergeMetadata",
```

Comment:
Nb: I don't think this very system property will be ported to NX3, so the package change is non-issue IMHO.